subworkflows/nf-core/fasta_binning_concoct/main.nf

```diff
@@ -0,0 +1,56 @@
+include { CONCOCT_CUTUPFASTA           } from '../../../modules/nf-core/concoct/cutupfasta/main.nf'
+include { CONCOCT_CONCOCTCOVERAGETABLE } from '../../../modules/nf-core/concoct/concoctcoveragetable/main.nf'
+include { CONCOCT_CONCOCT              } from '../../../modules/nf-core/concoct/concoct/main.nf'
+include { CONCOCT_MERGECUTUPCLUSTERING } from '../../../modules/nf-core/concoct/mergecutupclustering/main.nf'
+include { CONCOCT_EXTRACTFASTABINS     } from '../../../modules/nf-core/concoct/extractfastabins/main.nf'
+
+workflow FASTA_BINNING_CONCOCT {
+
+    take:
+    ch_fasta // channel: [ val(meta), [ fasta ] ]       // raw contigs from assembly
+    ch_bam   // channel: [ val(meta), [ bam ], [bai]]   // bam files of original FASTQ Files mapped back to each contig. meta must correspond to ch_fasta
+
+    main:
+    ch_versions = Channel.empty()
+
+    // required to create bedfile due to coverage table
+    produce_bedfile = true
+
+    CONCOCT_CUTUPFASTA ( ch_fasta, produce_bedfile )
+    ch_versions = ch_versions.mix(CONCOCT_CUTUPFASTA.out.versions.first())
```

Comment:
Nitpicky, but not really sure if the `first` is needed anymore

Reply:
I think it's to reduce channel clutter, right? Don't need 100 `version.ymls` being staged in the work dir for dumpsoftwareversions?

Reply:
Oh is that why that's done? I though it was some special channel magic case. nvm then!